src/app/game-state/item-repo.service.ts

```diff
@@ -332,15 +343,20 @@ export class ItemRepoService {
       value: 15,
       description: 'Little green trees. A very healthy vegetable.',
       useLabel: 'Eat',
-      useDescription: 'Fills your belly and helps you be healthy.',
+      useDescription: 'Fills your belly and helps you be healthy and hardy.',
       useConsumes: true,
-      use: () => {
-        this.characterService.characterState.status.nourishment.value++;
+      use: (count?: number) => {
+        if (!count){
+          count = 1;
+        }
```

Comment:
```suggestion
      use: (count = 1) => {
```

Reply:
K.. There's too many, you do the rest please!

Reply:
Interface doesn't allow default parameters.